bluemira/codes/process/params.py

```diff
@@ -325,11 +326,40 @@ class ProcessSolverParams(MappedParameterFrame):
     Z_eff: Parameter[float]
     """Effective particle radiation atomic mass [unified_atomic_mass_unit]."""
 
-    _mappings = None
+    _mappings = deepcopy(mappings)
+    _defaults = ProcessInputs()
 
     @property
     def mappings(self) -> Dict[str, ParameterMapping]:
         """Define mappings between these parameters and PROCESS's."""
-        if self._mappings is None:
-            self._mappings = deepcopy(mappings)
         return self._mappings
+
+    @property
+    def defaults(self) -> ProcessInputs:
+        """
+        Default values for Process
+        """
+        return self._defaults
```

Comment:
If these are edited, are they edited on the class? Do we need a copy, or, like I mentioned in a comment on the base class, could return a dictionary?